src/services/api-rest/identity-resolution/__tests__/identity-resolution.service.spec.ts

```diff
@@ -0,0 +1,172 @@
+import { ServiceUnavailableException } from '@nestjs/common';
```

Comment:
we need a general discussion of the endpoints that are exposed by the server microservice.